tests/expectations/test_expectation_arguments.py

```diff
@@ -591,6 +617,7 @@ def test_result_format_configured_no_set_default_override(
 
     result = results[0]
     assert len(result.result.keys()) == 0
+    assert result.result == {}
```

Comment:
For this one, the assertion as an empty dict looks a little strange. Could you demonstate that `result` has the expected boolean?

Reply:
@talagluck The empty `dict` in `result.result` is correct for `BOOLEAN_ONLY`.  Just added the assertion on the `boolean` flag (committed).  Please note that, importantly, the `result` object contains the auxiliary information (`ExpectationValidationResult`), while the `result.result` is a dictionary containing the detailed information about the validation.  In case `result_format` is `COMPLETE`, this dictionary is long; in case `result_format` is `SUMMARY` or `BASIC`, this dictionary is smaller, and in case `result_format` is `BOOLEAN_ONLY`, this dictionary is empty.  Thanks.

Reply:
Thanks, Alex! One last request - can we have a test asserting that `result` is equal to its expected value? It would be great to have a clear example in our tests of what we expect a `BOOLEAN_ONLY` result to look like, in case we need to point users to this. This will ensure that your explanation here is clear from the test case.

Reply:
@talagluck Great idea -- adding (in next commit).

Reply:
@talagluck Changes committed.  Thanks.